src/applications/vaos/components/layout/DetailPageLayout.jsx

```diff
@@ -202,31 +212,176 @@ function CancelButton({ appointment }) {
   return null;
 }
 
+export function TravelReimbursement({
+  appointmentDate,
+  claimData,
+  daysRemainingToFileClaim,
+  isPastAppointment,
+}) {
+  if (!isPastAppointment) return null;
+  if (!claimData) return null;
+
+  let body = null;
+
+  if (
+    claimData.message === TRAVEL_CLAIM_MESSAGES.noClaim &&
+    daysRemainingToFileClaim > 0
+  ) {
+    body = (
+      <>
+        <p className="vads-u-margin-y--0p5">
+          Days left to file: {daysRemainingToFileClaim}
+        </p>
+        <p className="vads-u-margin-y--0p5">
+          <va-link
+            data-testid="file-claim-link"
+            className="vads-u-margin-y--0p5"
+            href={`/appointments/claims/?date=${appointmentDate}`}
+            text="File a travel reimbursement claim"
+          />
+        </p>
+      </>
+    );
+  }
+  if (
+    claimData.message === TRAVEL_CLAIM_MESSAGES.noClaim &&
+    daysRemainingToFileClaim < 1
+  ) {
+    body = (
+      <>
+        <p className="vads-u-margin-y--0p5">
+          Days left to file: {daysRemainingToFileClaim}
+        </p>
+        <p className="vads-u-margin-y--0p5">
+          You didn’t file a claim for this appointment. You can only file for
+          reimbursement within 30 days of the appointment.
+        </p>
+        <p className="vads-u-margin-y--0p5">
+          <va-link
+            data-testid="how-to-file-claim-link"
+            className="vads-u-margin-y--0p5"
+            href="https://www.va.gov/resources/how-to-file-a-va-travel-reimbursement-claim-online/"
+            text="Learn more about travel reimbursement"
+          />
+        </p>
+      </>
+    );
+  }
+  if (claimData.id) {
+    body = (
+      <>
+        <p className="vads-u-margin-y--0p5">
+          You've already filed a claim for this facility and date.
+        </p>
+        <p className="vads-u-margin-y--0p5">
+          <va-link
+            data-testid="view-claim-link"
+            href={`/appointments/claims/${claimData.id}`}
+            text="Check your claim status"
+          />
+        </p>
+      </>
+    );
+  }
+
+  if (!body) return null;
+
+  return <Section heading="Travel reimbursement">{body}</Section>;
```

Comment:
nit: Instead of only conditionally setting the body and then returning a section here, just return the entire formatted section in the if statements, especially since the if statements are already returning `<>...</>` already. The current code looks unnecessarily complex.

Reply:
Something like this:

```
if(claimData.id) {
   return (<Section>...</Section>
}

// Question: Can claimData.message be something else other than noClaim? If so, how is that processed?
if(daysRemaningToFileClaim) {
   return (<Section>...</Section>
}

if(daysRemainingToFileClaim < 1) {
   return (<Section>...</Section>
}

// Default
return null;
```